### ISSUE TITLE
[onnx] I get many files when covert torch-model to onnx

### ISSUE THREAD

**Issue by Tian14267** [pytorch/pytorch#118372]
### 🚀 The feature, motivation and pitch

When I vovert my torch model to onnx, I get many files like this

![image](https://github.com/pytorch/pytorch/assets/27938135/cbe3b7bb-a2e1-489d-b2b6-9a3194bd49c9)
![image](https://github.com/pytorch/pytorch/assets/27938135/267432a0-f24a-4dbc-9bef-df7ccf6672de)

I'm confused. Why not output a onnx-model.  Why get too many files, just like split graph and weights in diffent file.

### Additional context

Here is my code

`model_path = "./saved_models/emotion_only_text_models_8k/"
	config = BertConfig.from_json_file(model_path)  # 加载bert模型配置信息
	config.num_labels=3

	device = 'cuda'
	input_ids_np = torch.from_numpy(np.zeros([1, 128], dtype=np.int64))
	token_type_ids_np = torch.from_numpy(np.zeros([1, 128], dtype=np.int64))
	attention_mask_np = torch.from_numpy(np.zeros([1, 128], dtype=np.int64))
	input_ids_tf = input_ids_np.type(torch.int64).to(device)
	token_type_ids_tf = token_type_ids_np.type(torch.int64).to(device)
	attention_mask_tf = attention_mask_np.type(torch.int64).to(device)
	feature_wav = torch.from_numpy(np.array(np.zeros((1,313, 128, 1))))
	feature_wav_tf = feature_wav.type(torch.float32).to(device)

	model = EmotionModel.from_pretrained(model_path, config=config).to("cuda")
	model.load_state_dict(torch.load(os.path.join(model_path, "pytorch_model.bin"),map_location="cuda"))
	print("####  加载模型完成")
	model.eval()

	onnx_name = "./temp/temp.onnx"
	torch.onnx.export(model,  # model being run
					  (input_ids_tf,token_type_ids_tf,attention_mask_tf,feature_wav_tf),  # model input (or a tuple for multiple inputs)
					  onnx_name,  # where to save the model
					  opset_version=11,  # the ONNX version to export the model to
					  input_names=['input_ids', 'token_type_ids', 'attention_mask', 'feature_wav'],
					  output_names=['logits'],
					  dynamic_axes={"input_ids": {0: "batch_size"},  # 批处理变量
									"token_type_ids": {0: "batch_size"},
									"attention_mask": {0: "batch_size"},
									"feature_wav": {0: "batch_size"},}
					  )`

**Comment by tringwald** [comment]
Is your model by any chance larger than 2GB? I remember there being a limitation in protobuf, so we have to split it into multiple files.

**Comment by Tian14267** [comment]
> Is your model by any chance larger than 2GB? I remember there being a limitation in protobuf, so we have to split it into multiple files.

Yes，it is large than 2GB 。   But how can I use those multiple files with onnx ? I try but it not work.

**Comment by tringwald** [comment]
There should still be a main ONNX file (probably `temp.onnx` in your case), which you can load.

**Comment by thiagocrepaldi** [comment]
exactly, you have to use `temp.onnx` as your model. the other files are just the initializers. The onnx rutnime will know how to load them
